client/my-sites/post-type-list/index.jsx

```diff
@@ -201,5 +237,6 @@ export default connect( ( state, ownProps ) => {
 		posts: getSitePostsForQueryIgnoringPage( state, siteId, ownProps.query ),
 		isRequestingPosts: isRequestingSitePostsForQueryIgnoringPage( state, siteId, ownProps.query ),
 		lastPage,
+		totalPosts: getSitePostsFoundForQuery( state, siteId, ownProps.query ),
```

Comment:
While we are making changes here, how about making the form of all these props consistent:

```
lastPage: getSitePostsLastPageForQuery( state, siteId, ownProps.query ),
```

Reply:
As #18998 is currently written, `getSitePostsFoundForQuery` will no longer work in all-sites mode.  Let's make sure we fix that, whether here or in that PR (most likely by renaming this selector to `getPostsFoundForQuery` and making it work in all-sites mode too).